rules/manager.go

```diff
@@ -342,7 +342,7 @@ func (m *Manager) LoadGroups(
 						externalLabels,
 						externalURL,
 						!shouldRestore,
-						m.logger.With("alert", r.Alert),
+						m.logger.With("alert", r.Alert.Value),
```

Comment:
we discussed this IRL and `r.Alert.Value` is the name of the alert, not the value, which makes sense to have in logs.